projects/js-packages/components/components/toggle-control/index.tsx

```diff
@@ -36,7 +33,6 @@ const ToggleControl: React.FC< ToggleControlProps > = ( {
 	help,
 	toggling,
 	label,
-	size = 'normal',
```

Comment:
Would either need to continue supporting but somehow deprecate, or remove now already, and run a codemode to remove the prop everywhere where the component is used with something like https://github.com/facebook/jscodeshift